Implemented configuration block to access other setting like font scale.

DastanIqbal opened the pull request:
This PR help devs to change configuration setting along with locale. For eg. Font Scale, uiMode, colorMode etc

Commit:
Implemented configuration block
Bump version

DastanIqbal committed:
Update README.md

gunhansancar commented:
Hi @DastanIqbal thank you for your contribution. It looks great. 
However, I'm not sure if this library should allow/enable such configuration changes. 
Because IMO this library is only about locale changes, nothing more and nothing less.

DastanIqbal commented:
Okay not an issues but when I used the library, then I couldn't change other setting and its restricted to library, because the block I have added that should run along with locale setting, otherwise changes won't reflect in app, like I added accessiblity features where I need to increase/decrease app text size, that wasn't working, because of the limitation, and  block should run before setting `createConfigurationContext` and its blocks developer.

gunhansancar commented:
Similar issues also arise about the dark mode, so I will accept the PR with some modifications, I'll send a commit on top of your PR. I should make it dynamically changeable as well like on button click etc.

DastanIqbal commented:
Cool Thanks

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Implemented accessibility font size

Commit:
Merge remote-tracking branch 'origin/master'

DastanIqbal commented:
I have fixed conflicts and added Accessibility Sample for font Size in MainActivity, 
currently, I am using forked in my app. because we can't change font size in the current implementation

thubalek commented on .idea/misc.xml:
```diff
@@ -5,7 +5,6 @@
       <configuration PROFILE_NAME="Debug" CONFIG_NAME="Debug" />
     </configurations>
   </component>
-  <component name="ExternalStorageConfigurationManager" enabled="true" />
```
Is this change part of the pull request intentionally? Is it somehow related to implementation?

DastanIqbal replied:
No, It shouldn't be, let me fix the conflicts

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	build.gradle
#	localehelper/src/main/java/com/zeugmasolutions/localehelper/LocaleHelper.kt

Commit:
added missing configuration line ExternalStorageConfigurationManager
